mscore/file.cpp

```diff
@@ -1209,9 +1209,29 @@ QString MuseScore::getFotoFilename(QString& filter, QString* selectedFilter)
       // setup side bar urls
       saveImageDialog->setSidebarUrls(urls);
 
+      // if no saves were made for current score, then use the score's name as default
+      if(cs->masterScore()->savedCapture() == false) {
```

Comment:
Could you fix that and squash your commit http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html